### ISSUE TITLE
Setup virtual env for any python scripts

### ISSUE THREAD

**Issue by chatton** [cosmos/ibc-go#3837]
## Summary

There has been work done recently on adding a [python script](https://github.com/cosmos/ibc-go/pull/3541).

In order to make things consistent between environments, I suggest we setup a [virtual environment](https://docs.python.org/3/library/venv.html).

We could add a Makefile target like

`python-init`

`compatibility-tests: python-init`

To ensure an environment before running the script.

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged/assigned

**Comment by DimitrisJim** [comment]
and lets point out that it isn't currently required since the script works with any version of Python >= 3.9 and uses no third-party dependencies.

Would make developing something that would require 3rd party deps a much smoother process.

**Comment by chatton** [comment]
Yes! Good point @DimitrisJim , we don't need to worry about this until we have dependencies, (like a [formatter/linter!](https://github.com/cosmos/ibc-go/issues/3838))
